exp-player/addon/components/exp-player.js

```diff
@@ -148,6 +148,9 @@ able to continue the study.
                 this._transition();
                 this.set('frameIndex', frameIndex + 1);
                 this.set('framePage', 0);
+                this.get('session').set('frameIndex', frameIndex + 1);
+                this.get('session').set('surveyPage', 0);
+                this.get('session').save();
```

Comment:
I think session may already be saved somewhere else, whenever we advance a frame. Is this causing data to be saved twice for each click of "next"?